packages/server/src/common/migrations/1760649528955-InitialSetup.ts

```diff
@@ -31,14 +33,25 @@ export class InitialSetup1760649528955 implements MigrationInterface {
       ],
     });
 
+    const councilsTable = new Table({
+      name: this.COUNCILS_TABLE,
+      columns: [
+        { name: 'id', type: 'int', isPrimary: true, isGenerated: true, generationStrategy: 'increment' },
+        { name: 'name', type: 'varchar', isNullable: false },
+        { name: 'lat', type: 'double precision', isNullable: false },
+        { name: 'lon', type: 'double precision', isNullable: false },
+      ],
+      indices: [{ columnNames: ['name'], isUnique: false }],
+    });
+
     const eventsTable = new Table({
       name: this.EVENTS_TABLE,
       columns: [
         { name: 'id', type: 'uuid', isPrimary: true, isUnique: true, isNullable: false },
         { name: 'title', type: 'varchar', isNullable: false },
         { name: 'description', type: 'text', isNullable: false },
         { name: 'date', type: 'timestamptz', isNullable: false },
-        { name: 'council_id', type: 'uuid', isNullable: false },
+        { name: 'council_id', type: 'int', isNullable: false },
```

Comment:
UUID's are preferable to integers for ID's as discussed in the comments on the PR.